src/configuration/app/storage.md

```diff
@@ -114,6 +114,17 @@ web:
 
 See the [web locations](/configuration/app/web.md) documentation for more details.
 
+## Unable to mount hidden folder?
+
+We ignore every YAML key that starts with a dot. This causes a mount like '.myhiddenfolder' to be ignored. If you want to mount a hidden folder, you'll have to prepend it with a `/`
+
+```yaml
+mounts:
+  '/.myhiddenfolder':
+    source: local
+    source_path: '.myhiddenfolder'
```

Comment:
```suggestion
    source_path: 'myhiddenfolder'
```

Reply:
This one's your call. In our templates we don't keep mount source paths hidden to keep it simple and visible, so this suggestion is more of keeping it consistent with that choice.

Reply:
Dot files are annoying, so let's not use them in source paths.